pkgs/tools/misc/cunit/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchurl, autoconf, automake, libtool, autoreconfHook}:
 
-stdenv.mkDerivation rec {
```

Comment:
We should probably split out such bigger changes to make them easy to review and get them in faster than in this bigger PR.

Reply:
ok im going to split this up